examples/flask-kitchensink/app.py

```diff
@@ -51,11 +51,8 @@
 # get channel_secret and channel_access_token from your environment variable
 channel_secret = os.getenv('LINE_CHANNEL_SECRET', None)
 channel_access_token = os.getenv('LINE_CHANNEL_ACCESS_TOKEN', None)
-if channel_secret is None:
-    print('Specify LINE_CHANNEL_SECRET as environment variable.')
-    sys.exit(1)
-if channel_access_token is None:
-    print('Specify LINE_CHANNEL_ACCESS_TOKEN as environment variable.')
+if channel_secret is None or channel_access_token is None:
```

Comment:
The previous version looks better to me.

Reply:
It's because it only said "Specify LINE_CHANNEL_SECRET" even when both of them were not specified.
Thanks for merging it anyway :)